overcooked_ai_py/mdp/overcooked_mdp.py

```diff
@@ -884,6 +884,19 @@ def _check_valid_state(self, state):
         for obj_state in all_objects:
             assert obj_state.is_valid()
 
+    def find_free_counters_valid_for_both_players(self, state, mlp):
+        """Finds all empty counter locations that are accessible to both players"""
+        one_player = state.players[0]
+        other_player = state.players[1]
+        free_counters = self.get_empty_counter_locations(state)
+        free_counters_valid_for_both = []
+        for free_counter in free_counters:
+            goals = mlp.mp.motion_goals_for_pos[free_counter]
+            if (True in [mlp.mp.is_valid_motion_start_goal_pair(one_player.pos_and_or, goal) for goal in goals]) and \
```

Comment:
`any([mlp.mp.is_valid_motion_start_goal_pair(one_player.pos_and_or, goal) for goal in goals])` achieves the same thing as `True in ...` and is more elegant